test/test_tensordict.py

```diff
@@ -1189,6 +1189,13 @@ def test_view(self, td_name, device):
         assert (td_view.get("a") == 1).all()
         assert (td.get("a") == 1).all()
 
+    def test_default_nested(self, td_name, device):
+        torch.manual_seed(1)
+        td = getattr(self, td_name)(device)
+        default_val = torch.randn(())
+        timbers = td.get(("shiver", "my", "timbers"), default_val)
```

Comment:
I like the keys you come up with for tests 😛

Reply:
just to check that my code is properly reviewed
I can check that box now!